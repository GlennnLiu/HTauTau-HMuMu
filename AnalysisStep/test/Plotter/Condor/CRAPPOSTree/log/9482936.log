000 (9482936.006.000) 07/27 15:12:38 Job submitted from host: <137.138.120.116:9618?addrs=137.138.120.116-9618+[2001-1458-d00-16--17b]-9618&alias=bigbird12.cern.ch&noUDP&sock=schedd_8624_ecd7>
...
040 (9482936.006.000) 07/27 15:14:46 Started transferring input files
	Transferring to host: <128.142.61.36:9618?addrs=128.142.61.36-9618+[2001-1458-301-23--100-23]-9618&alias=b7s06p7054.cern.ch&noUDP&sock=slot1_19_4260_4e26_174182>
...
040 (9482936.006.000) 07/27 15:14:46 Finished transferring input files
...
001 (9482936.006.000) 07/27 15:14:47 Job executing on host: <128.142.61.36:9618?addrs=128.142.61.36-9618+[2001-1458-301-23--100-23]-9618&alias=b7s06p7054.cern.ch&noUDP&sock=startd_2986_b9e2>
...
006 (9482936.006.000) 07/27 15:14:55 Image size of job updated: 584148
	92  -  MemoryUsage of job (MB)
	94068  -  ResidentSetSize of job (KB)
...
040 (9482936.006.000) 07/27 15:15:03 Started transferring output files
...
040 (9482936.006.000) 07/27 15:15:03 Finished transferring output files
...
005 (9482936.006.000) 07/27 15:15:03 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:04, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:04, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	24044  -  Run Bytes Sent By Job
	346  -  Run Bytes Received By Job
	24044  -  Total Bytes Sent By Job
	346  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1         1 
	   Disk (KB)            :       38        1   1311425 
	   Memory (MB)          :       92     2000      2000 

	Job terminated of its own accord at 2023-07-27T13:15:03Z.
...
