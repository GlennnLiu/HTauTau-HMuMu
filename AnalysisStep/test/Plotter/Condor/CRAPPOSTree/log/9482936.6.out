[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.diBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.triBoson.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.top.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.EWKZee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.EWKZtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.EWKZmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.DYee.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.DYtt.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.DYmm.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.Wjet.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.TTfake.root
[INFO] Get histos from /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.Data.root
[INFO] Total MC summed
[INFO] Saved Histos to /eos/home-g/geliu/LepUni/Histos/UL2016/DistributionPlots/CRAPPOSTree.TotalMC.root
